libtransmission/port-forwarding.cc

```diff
@@ -92,7 +93,12 @@ static void natPulse(tr_shared* s, bool do_check)
     {
         session->public_peer_port = public_peer_port;
         session->private_peer_port = received_private_port;
-        loginfo("public peer port %d (private %d) ", session->public_peer_port, session->private_peer_port);
+        tr_logAddNamedInfo(
+            CodeName,
+            fmt::format(
+                _("Mapped private port '{private_port}' to public port '{public_port}'"),
```

Comment:
Sorry to bug again but I really think we need a common style for those placeholder names underscore- and contraction-wise. Happy to delay this for the final stroke in this patch series though.

Reply:
Updated in 4a36a2014db65b5565fd768c2f7240893910e05d. 

As of the latest commit (5fb750bec095b0138136746ab15c3154a017d22c), the placeholder list is:

```
actual_size
address
bundle
content_type
count
crypto_library
entry
error
error_code
expected_size
file_size
host
key
line
mode
old_path
old_state
path
port
pos
prefix
private_port
public_port
size_requested
size_used
socket
state
text
type
url
version

```